.github/workflows/tests.yaml

```
@@ -0,0 +1,32 @@
+name: Tests
+run-name: "Tests on ${{ github.ref_name }}: ${{ github.sha }}"
+on:
+  push: { }
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Node.js
```

actionlint (1.7.4)

"uses" is required to run action in step

(syntax-check)